merge.go

```diff
@@ -139,7 +139,23 @@ func Merge(memGraph *lpg.Graph, dbGraph *DBGraph, config Config) ([]Delta, error
 				}
 			}
 		}
-		deltas = mergeSubtree(n, foundDBEntity, dbGraph, nodeAssociations, edgeAssociations, deltas)
+		sfx := strings.LastIndex(schemaPV.AsString(), "/")
+		if sfx != -1 {
+			if op, exists := entityActions[schemaPV.AsString()[:sfx]]; exists {
+				// [merge, nocreate]
+				f := strings.Fields(op)
+				if f[0] == "merge" && f[1] == "create" {
```

Comment:
If merge && create
   call merge
if merge && !create 
   call merge if dbGraph is nonnill
if !merge && create
   call merge if dbgraph is nil
if !merge && !create
   don't call merge